.github/CODEOWNERS

```
@@ -45,7 +45,7 @@
 /charts/prometheus-rabbitmq-exporter/ @desaintmartin @iamabhishek-dubey @juanchimienti @monotek
 /charts/prometheus-redis-exporter/ @acondrat @zanhsieh
 /charts/prometheus-smartctl-exporter/ @kfox1111 @zeritti
-/charts/prometheus-snmp-exporter/ @miouge1 @xiu
+/charts/prometheus-snmp-exporter/ @miouge1 @xiu @walker-tom
```

This needs to be sorted.

```suggestion
/charts/prometheus-snmp-exporter/ @miouge1 @walker-tom @Nexucis 
```